registration/models.py

```diff
@@ -49,7 +52,63 @@ class OKUser(AbstractUser):
     # If a User specifies an email address it needs to be unique
     username = None
     email = models.EmailField(
-        _('email address'), blank=True, unique=True, null=True)
+        _('email address'), unique=True, null=True)
     USERNAME_FIELD = 'email'
     REQUIRED_FIELDS = []
     objects = UserManager()
+
+
+class Profile(models.Model):
+    """
+    Model for a profil.
+
+    A profil stores further informations about a user. To every profil
+    belongs a user.
+    """
+
+    # TODO auch so, dass kein OKUser existiert
+    okuser = models.OneToOneField(OKUser, on_delete=models.CASCADE)
+
+    first_name = models.CharField(blank=False, null=True, max_length=150)
+    last_name = models.CharField(blank=False, null=True, max_length=150)
+
+    class Gender(models.TextChoices):
+        """The gender of the user."""
+
+        NOT_GIVEN = 'none', _('Not Given')
+        MALE = 'm', _('Male')
+        FEMALE = 'f', _('Female')
+        DIVERSE = 'd', _('Diverse')
+
+    gender = models.CharField(
+        max_length=4,
+        choices=Gender.choices,
+        default=Gender.NOT_GIVEN,
+    )
+
+    # TODO uses phonenumber-field
+    # https://pypi.org/project/django-phonenumber-field/
+
```

Comment:
Das kann eigentlich weg
```suggestion
    # TODO uses phonenumber-field
    # https://pypi.org/project/django-phonenumber-field/

```